components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterConstants.java

```diff
@@ -99,7 +105,9 @@ public enum ErrorMessages {
         ERROR_CREATING_SSL_CONTEXT("65007", "Error while preparing SSL context for WebSubHub http client.",
                 "Server error encountered while preparing SSL context for WebSubHub http client."),
         ERROR_CREATING_ASYNC_HTTP_CLIENT("65008", "Error while creating the Async HTTP client.",
-                "Server error encountered while creating the Async HTTP Client of WebSub Hub Adapter.");
+                "Server error encountered while creating the Async HTTP Client of WebSub Hub Adapter."),
+        TOPIC_DEREGISTRATION_FAILURE_ACTIVE_SUBS("65016", "Error occurred while de-registering topic", "Backend error" +
```

Comment:
Any particular reason to use 65016 instead of 65009?

Reply:
Resolves by 6717b14f2cb19b0d561cda6b124c16cfbbe605d2